src/sites/doc/docs/react/intro-react.en-US.md

```
@@ -0,0 +1,42 @@
+- 🍭 Unit test coverage exceeds 80% to ensure stability
+- 📖 Sketch design resources are available
+
+## Examples
+
+<img src="https://storage.360buyimg.com/imgtools/cc08cb2760-d70f19c0-0a95-11ee-808d-b1254110adba.png" width="200" alt="NutUI-React" />
+
+## Compatibility Environment
+
+| Version | Supported Frameworks | Visual Specifications | Compatibility |
+| --- | --- | --- | --- |
+| @nutui/nutui-react <img src="https://img.shields.io/npm/v/@nutui/nutui-react" /> | React 17\18 | Jingdong [APP 10.0](#/resource) | Modern browsers start with Chrome >= 51、iOS >= 10.0 |
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)